src/ray/object_manager/plasma/store.cc

```diff
@@ -554,8 +556,15 @@ void PlasmaStore::PrintAndRecordDebugDump() const {
 }
 
 void PlasmaStore::RecordMetrics() const {
-  // TODO(sang): Add metrics.
+  absl::MutexLock lock(&mutex_);
+  RAY_LOG(DEBUG) << "Exporting plasma store metrics.";
   object_lifecycle_mgr_.RecordMetrics();
+
+  // Schedule the next metric export callback
+  metrics_timer_ = execute_after(
+      io_context_,
+      [this]() { RecordMetrics(); },
```

Comment:
Why don't we just call it here? I think `PlasmaStore::RecordMetrics()` must be already periodically called?

Reply:
It was priodically called in `PrintAndRecordDebugDump` which has a similiar timer setup.